Pear/Onboarding/GoalsViewController.swift

```diff
@@ -149,20 +150,15 @@ class GoalsViewController: UIViewController {
     }
 
     @objc func nextButtonPressed() {
-        NetworkManager.shared.updateUserGoals(goals: selectedGoals).observe { [weak self] result in
+        Networking2.updateGoals(goals: selectedGoals) { [weak self] (success) in
             guard let self = self else { return }
-                switch result {
-                case .value(let response):
-                    DispatchQueue.main.async {
-                        if response.success {
-                            self.delegate?.nextPage(index: 4)
-                        } else {
-                            self.present(UIAlertController.getStandardErrortAlert(), animated: true, completion: nil)
-                        }
-                    }
-                case .error:
+            DispatchQueue.main.async {
+                if success {
+                    self.delegate?.nextPage(index: 4)
```

Comment:
i saw a nextPage with index 1 earlier and misinterpreted it as going to the next page one by one but I can see that's not the case - an enum here would probably make it clearer which index is what